networking_ccloud/tools/netbox_config_gen.py

```diff
@@ -296,6 +329,7 @@ def handle_interface_or_portchannel(cls, iface: NbRecord, candidate_interfaces:
                                         switchport_kwargs: Dict[str, Any] = {}):
 
         device_name = iface.device.name  # type: ignore
+        speed = cls._handle_interface_speed(iface)
```

Comment:
even if it's a class method I'd use `self` here to call it.

Reply:
I am calling it from a classmethod. How'd I have `self` available?

Reply:
Whops, you wouldn't of course. Seems like I looked wrong.